test/graph/graph-mary-manipulation-test.js

```diff
@@ -8,27 +8,34 @@ var maryStream = testStreams.fullList.filter(function(stream) {
 
 describe('Graph manipulation for `mary` stream', function() {
     var graph;
-    beforeEach(function(done) {
+    function createGraph(done) {
```

Comment:
The logic of graph.createNode() changed, and it's no longer a "clean" function call (the node returned is already added in the graph). By moving graph creation into the nested describe blocks we get a clean graph for every it() block test.